plugin.go

```diff
@@ -49,6 +53,11 @@ func (p *BasePlugin) GraphqlQueryPath() (bool, string) {
 	return false, ""
 }
 
+// InterceptRequest is called before bramble starts executing a request.
+// It can be used to inspect the unmarshalled GraphQL request bramble receives.
+func (p *BasePlugin) InterceptRequest(ctx context.Context, operationName, rawQuery string, variables map[string]interface{}) {
```

Comment:
I might have suggested naming this something like `WrapClientRequest` because it's on the external-client-side of a Bramble instance.